### PR TITLE
Add metering point client for bff

### PR DESCRIPTION
**Pull request by thbaan**
## References

* #0000

### CODE REVIEW

**Review by dstsyst — DISMISSED** [review]

**Review by MrDach — APPROVED** [review]

File: source/Energinet.DataHub.MeteringPoints.Client.Abstractions/Models/MeteringPointDto.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        string MeterNumber,
+        string StreetCode,
+        string CitySubDivisionName,
+        string Floor,
+        string Suite,
+        string BuildingNumber,
+        int MunicipalityCode,
+        bool IsActualAddress,
+        Guid? GeoInfoReference,
+        double Capacity,
```
Comment (erduna) [inline source/Energinet.DataHub.MeteringPoints.Client.Abstractions/Models/MeteringPointDto.cs:48]: Can be null

File: source/Energinet.DataHub.MeteringPoints.Client.Abstractions/Models/MeteringPointDto.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        string Product,
+        string UnitType,
+        DateTime? EffectiveDate,
+        string MeterNumber,
+        string StreetCode,
+        string CitySubDivisionName,
+        string Floor,
+        string Suite,
+        string BuildingNumber,
+        int MunicipalityCode,
```
Comment (erduna) [inline source/Energinet.DataHub.MeteringPoints.Client.Abstractions/Models/MeteringPointDto.cs:45]: Can be null for special MPs

File: source/Energinet.DataHub.MeteringPoints.Client.Abstractions/Models/MeteringPointDto.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        string StreetName,
+        string PostCode,
+        string CityName,
+        string CountryCode,
+        string ConnectionState,
+        string MeteringPointSubType,
+        string ReadingOccurrence,
+        string MeteringPointType,
+        int MaximumCurrent,
+        int MaximumPower,
```
Comment (erduna) [inline source/Energinet.DataHub.MeteringPoints.Client.Abstractions/Models/MeteringPointDto.cs:31]: Can be null

File: source/Energinet.DataHub.MeteringPoints.Client.Abstractions/Models/MeteringPointDto.cs [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        string GsrnNumber,
+        string StreetName,
+        string PostCode,
+        string CityName,
+        string CountryCode,
+        string ConnectionState,
+        string MeteringPointSubType,
+        string ReadingOccurrence,
+        string MeteringPointType,
+        int MaximumCurrent,
```
Comment (erduna) [inline source/Energinet.DataHub.MeteringPoints.Client.Abstractions/Models/MeteringPointDto.cs:30]: Can be null

File: source/Energinet.DataHub.MeteringPoints.Client.Abstractions/Models/MeteringPointDto.cs [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        string UnitType,
+        DateTime? EffectiveDate,
+        string MeterNumber,
+        string StreetCode,
+        string CitySubDivisionName,
+        string Floor,
+        string Suite,
+        string BuildingNumber,
+        int MunicipalityCode,
+        bool IsActualAddress,
```
Comment (erduna) [inline source/Energinet.DataHub.MeteringPoints.Client.Abstractions/Models/MeteringPointDto.cs:46]: Never present for special MPs, so should be nullable.

File: source/Energinet.DataHub.MeteringPoints.Client.Abstractions/Models/MeteringPointDto.cs [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        double Capacity,
+        string SettlementMethod,
+        string NetSettlementGroup,
+        string AssetType,
+        string ToGridAreaCode,
+        string FromGridAreaCode,
+        DateTime? SupplyStart,
+        string ConnectionType,
+        string DisconnectionType,
+        bool ProductionObligation);
```
Comment (erduna) [inline source/Energinet.DataHub.MeteringPoints.Client.Abstractions/Models/MeteringPointDto.cs:57]: Only present for productin MPs, so should be nullable

File: source/Energinet.DataHub.MeteringPoints.IntegrationTests/Client/MeteringPointClientTests.cs [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+        private readonly WebApiFactory _factory;
+
+        public MeteringPointClientTests(WebApiFactory factory, DatabaseFixture databaseFixture)
+            : base(databaseFixture)
+        {
+            _factory = factory;
+        }
+
+        [Fact]
+        public async Task Get_metering_point_by_id_should_not_be_null()
```
Comment (erduna) [inline source/Energinet.DataHub.MeteringPoints.IntegrationTests/Client/MeteringPointClientTests.cs:37]: should this be named "Get_metering_point_by_gsrn_should_not_be_null ?

File: source/Energinet.DataHub.MeteringPoints.ApplyDBMigrationsApp/Scripts/Model/202105181215 Create MeteringPoints tables.sql [hunk 8]
```diff
@@ -42,7 +42,6 @@ CREATE TABLE [dbo].[ConsumptionMeteringPoints]
 (
     [Id] UNIQUEIDENTIFIER NOT NULL,
     [RecordId] INT IDENTITY(1,1) NOT NULL,
-    [IsAddressWashable] BIT NOT NULL DEFAULT(0),
```
[the remark this answers is not in the document]
Comment (thbaan) [reply source/Energinet.DataHub.MeteringPoints.ApplyDBMigrationsApp/Scripts/Model/202105181215 Create MeteringPoints tables.sql:45]: Not really. It´s just a column which is not in use. It´s not strictly necessary to remove it from existing environments, but it definitely doesn't make sense deploying it to a new environment.

**Review by cmgrd — APPROVED** [review]

**Review by Korgath — APPROVED** [review]

**Pull request merged**